cli/initiator/reshare.go

```diff
@@ -79,21 +76,31 @@ var StartReshare = &cobra.Command{
 		if ethNetwork == "" {
 			logger.Fatal("😥 Cant recognize eth network")
 		}
-		// Open ethereum keystore
-		jsonBytes, err := os.ReadFile(cli_utils.KeystorePath)
+		signatures, err := cli_utils.SignaturesStringToBytes(cli_utils.Signatures)
 		if err != nil {
-			return err
+			logger.Fatal("😥 Failed to load signatures: ", zap.Error(err))
 		}
-		keyStorePassword, err := os.ReadFile(filepath.Clean(cli_utils.KeystorePass))
+		// Contruct the resign message
+		rMsg, err := dkgInitiator.ConstructReshareMessage(
+			oldOperatorIDs,
+			newOperatorIDs,
+			signedProofs[0][0].Proof.ValidatorPubKey,
+			ethNetwork,
+			cli_utils.WithdrawAddress[:],
+			cli_utils.OwnerAddress,
+			cli_utils.Nonce,
+			signedProofs[0],
+		)
```

Comment:
Indeed in this cli users pass the data for generating the reshare/resign messages instead of the messages themselves. Everything can be a string now including the `signedProofs` (updated in the bulk PR), so from the users perspective it would be pretty much the same. But of course we can update this if required.